core/math/a_star.h

```diff
@@ -31,9 +31,20 @@
 #ifndef A_STAR_H
 #define A_STAR_H
 
+#include "core/math/math_defs.h"
+#include "core/math/vector2.h"
+#include "core/math/vector3.h"
 #include "core/object/gdvirtual.gen.inc"
+#include "core/object/object.h"
 #include "core/object/ref_counted.h"
+#include "core/os/memory.h"
+#include "core/string/string_name.h"
+#include "core/templates/hash_set.h"
 #include "core/templates/oa_hash_map.h"
+#include "core/templates/pair.h"
+#include "core/templates/vector.h"
+#include "core/typedefs.h"
+#include "core/variant/variant.h"
```

Comment:
Here, for example, this doesn't affect anything, as `variant.h` is already included via `object.h`, meaning that it'll just enter the file, hit the include guard, and exit again